build-tests/api-extractor-test-04/etc/api-extractor-test-04.api.md

```diff
@@ -60,11 +60,15 @@ export namespace EntangledNamespace {
 // @alpha
 export type ExportedAlias = AlphaClass;
 
+// Warning: (ae-internal-missing-underscore) The name InternalClass should be prefixed with an underscore because the declaration is marked as "@internal"
+// 
```

Comment:
In this case the warning is associated with the class declaration, since it uses an inline export.